Assignments/SCF-Session-3/area/Area.java

```diff
@@ -0,0 +1,111 @@
+package area;
+
+import java.util.Scanner;
+
+/**
+ * Calculates area of different shapes.
+ * @author Ishita Mathur
+ */
+class shapesArea
+{
+	/**
+	 * Calculates the area of triangle
+	 * @param width, width > 0
+	 * @param height, height > 0
+	 * @return area of triangle
+	 */
+	public double triangleArea(double width, double height) throws ArithmeticException
+	{
+		return 0.5 * width * height;
+	}
+	
+	/**
+	 * Calculates the area of rectangle
+	 * @param width, width > 0
+	 * @param height, height > 0
+	 * @return area of rectangle
+	 */
+	public double rectangleArea(double width, double height) throws ArithmeticException
+	{
+		return width * height;
+	}
+	
+	/**
+	 * Calculates the area of square
+	 * @param width, width > 0
+	 * @return area of square
+	 */
+	public double squareArea(double width) throws ArithmeticException
+	{
+		return width * width;
+	}
+	
+	/**
+	 * Calculates the area of circle
+	 * @param radius, radius > 0
+	 * @return area of circle
+	 */
+	public double circleArea(double radius) throws ArithmeticException
+	{
+		return 3.14 * radius * radius;
+	}
+}
+public class Area 
+{
+	public static void main(String[] args)
+	{
+		Scanner sc = new Scanner(System.in);
+		shapesArea area = new shapesArea();
+		int choice;
+		double width, height, radius;
+		do
+		{
+			System.out.println("MENU");
+			System.out.println("1. Area of Triangle");
```

Comment:
`\n` can be used to add new line instead of using multiple println statements.

Reply:
done